### PR TITLE
Simplified setup scripts for WSO2 IAM version 6.0.0

### PR DESCRIPTION
**Pull request by Anandi-K**

### CODE REVIEW

File: simple/deployment-scripts/wso2is-latest.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ license_text="LICENSE.txt"
 
 # wso2 image variables
 IMG_DEST="docker.wso2.com"
-IMG_TAG="5.11.0.0"
```
Comment (piraveena) [inline simple/deployment-scripts/wso2is-latest.sh:26]: change the copyright text to `WSO2 Inc` -> WSO2 LLC. Fix in all the places

File: simple/README.md [hunk 2]
```diff
@@ -33,12 +33,6 @@
 2. Move into the directory, where you have downloaded the aforementioned resources in step 1.
 
 3. Deploy WSO2 Identity Server in your Kubernetes cluster.
-
-    * Deploy WSO2 Identity Server using Docker images from DockerHub.
```
Comment (piraveena) [inline simple/README.md:37]: Remove these lines too 
  ```WSO2 product Docker images available at [DockerHub](https://hub.docker.com/u/wso2/) package General Availability (GA)
  versions of WSO2 products with no [WSO2 Updates](https://wso2.com/updates). ``` https://github.com/wso2/kubernetes-is/pull/323/files#diff-485807cbe3125f89817c993b149ea90950aa1a89c59c7eaf72c03eb6bf402de4R21

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,54 +1,11 @@
 # Changelog
 
-All notable changes to Kubernetes and Helm resources for WSO2 IAM version `5.11.x` in each resource release, will be documented in this file.
+All notable changes to Kubernetes and Helm resources for WSO2 IAM version `6.0.x` in each resource release, will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [v5.11.0.5] - 2021.11.25
+## [v6.0.0.1] - 2022-08-17
```
Comment (Buddhimah) [inline CHANGELOG.md:7]: Can we add to the changelog that we dont offer opensource image anymore?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,54 +1,11 @@
...
 
 ### Added
 
-- Integrate Kubernetes Startup Probe to Identity Server workload (refer [issue](https://github.com/wso2/kubernetes-is/issues/230))
-- Use WSO2 Identity Server Health Check API for Kubernetes Readiness Probes (refer [issue](https://github.com/wso2/kubernetes-is/issues/252))
-
-### Changed
-
-- Upgrade the API version of Kubernetes Ingress resource (refer [issue](https://github.com/wso2/kubernetes-is/issues/312))
```
Comment (Buddhimah) [inline CHANGELOG.md:16]: Also shouldnt we add about the user store change and the jdk support cc: @piraveena

Comment (piraveena) [reply CHANGELOG.md:16]: +1 to add.

File: simple/README.md [hunk 5]
```diff
@@ -33,12 +30,6 @@
...
-    
-        ```
-        ./wso2is-ga.sh --deploy
-        ```
```
Comment (piraveena) [inline simple/README.md:41]: shall we add readme on how to run the script?

**Review by piraveena — APPROVED** [review]

**Pull request merged**